test/hotspot/gtest/runtime/test_committed_virtualmemory.cpp

```diff
@@ -83,14 +83,21 @@ class CommittedVirtualMemoryTest {
     ASSERT_TRUE(found_stack_top);
   }
 
+  static constexpr int PAGE_CONTAINED_IN_RANGE_TAG = -1;
+  static constexpr bool IS_PAGE_CONTAINED_IN_COMMITTED_REGION(int a) { return (a == PAGE_CONTAINED_IN_RANGE_TAG); }
+  static constexpr void SET_PAGE_AS_CONTAINED_IN_COMMITTED_REGION(int &a) { a = PAGE_CONTAINED_IN_RANGE_TAG; }
```

Comment:
The functions aren't `constexpr`, so you can skip those. I think all of these can be in `this_style_of_naming` instead.